packages/commonwealth/server/models/contract.ts

```diff
@@ -0,0 +1,62 @@
+import * as Sequelize from 'sequelize'; // must use "* as" to avoid scope errors
+import { DataTypes } from 'sequelize';
+import { ChainNodeAttributes, ChainNodeInstance } from './chain_node';
+import { CommunityContractAttributes, CommunityContractInstance } from './community_contract';
+import { ModelStatic, ModelInstance } from './types';
+
+
+export type ContractAttributes = {
+  id: number;
+  address: string;
+  chain_node_id: number;
+  decimals?: number;
+  token_name?: string;
+  symbol?: string;
+  type: string;
```

Comment:
What are the possible types?

Reply:
currently, `types` are just our network from the chain model, what we use to identify the controller initialization steps. there will be a few more in the list once i get the solana tokens moved over but this this the current: 
```
 aave
 erc721
 compound
 marlin-testnet
 erc20
```

Reply:
Can we use an enum somewhere to make it clear what the list of valid types are?